doc/source/cluster/kubernetes/user-guides/gke-gcs-bucket.md

```diff
@@ -0,0 +1,168 @@
+# Configuring KubeRay to use Google Cloud Storage Buckets in GKE
+
+If you are already familiar with Workload Identity in GKE, you can skip this document.  The gist is that you need to specify a service account in each of the Ray pods after linking your Kubernetes service account to your Google Cloud service account.  Otherwise, read on.
+
+We will follow an abridged version of the documentation at <https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity>.  The full documentation is worth reading if you are interested in the details.
+
+## Create a Kubernetes cluster on GKE
+
+For this example, we will create a minimal KubeRay cluster using GKE.
+
+Run this command and all following commands on your local machine or on the [Google Cloud Shell](https://cloud.google.com/shell). If running from your local machine, you will need to install the [Google Cloud SDK](https://cloud.google.com/sdk/docs/install).
+
+```bash
+gcloud container clusters create cloud-bucket-cluster \
+    --num-nodes=1 --min-nodes 0 --max-nodes 1 --enable-autoscaling \
+    --zone=us-west1-b --machine-type e2-standard-8 \
+    --workload-pool=my-project-id.svc.id.goog # Replace my-project-id with your GCP project ID
+```
+
+
+This command creates a Kubernetes cluster named `cloud-bucket-cluster` with 1 node in the `us-west1-b` zone. In this example, we use the `e2-standard-8` machine type, which has 8 vCPUs and 32 GB RAM.
+
+Now get credentials for the cluster to use with `kubectl`:
+
+```bash
+gcloud container clusters get-credentials cloud-bucket-cluster --zone us-west1-b --project my-project-id
```

Comment:
Should we provide instructions about how to get `my-project-id`?

Reply:
To do anything in GCP the user has to create a project and project ID, so I think it's okay not to discuss it in the doc.

Reply:
Well it doesn't hurt, so I added a link